src/dmd/mars.d

```diff
@@ -2674,20 +2674,20 @@ Modules createModules(ref Strings files, ref Strings libmodules)
             }
             else
             {
-                error(Loc.initial, "unrecognized file extension %s", ext);
+                error(Loc.initial, "unrecognized file extension %.*s", ext.length, ext.ptr);
```

Comment:
The length is supposed to be of type `int`.

Reply:
So just add cast(int) ?

Reply:
Yes.